sunshine/httpcommon.cpp

```diff
@@ -94,8 +94,12 @@ int save_user_creds(const std::string &file, const std::string &username, const
   BOOST_LOG(info) << "New credentials have been created"sv;
 
   if(run_our_mouth) {
```

Comment:
With your latest work, creating a password automatically is no longer necessary.